src/main/java/de/tum/in/www1/artemis/web/rest/settings/ide/IdeDTO.java

```diff
@@ -0,0 +1,10 @@
+package de.tum.in.www1.artemis.web.rest.settings.ide;
+
+import de.tum.in.www1.artemis.domain.settings.ide.Ide;
+
+public record IdeDTO(String name, String deepLink) {
```

Comment:
DTOs usually reside in the `src/main/java/de/tum/in/www1/artemis/web/rest/dto` folder.

Reply:
is that the reason for the failing arch test as well?